src/widget/Mainframe.cpp

```diff
@@ -505,6 +523,10 @@ void Mainframe::closeEvent(QCloseEvent* event) {
 }
 
 void Mainframe::open() {
+  this->open(nullptr);
```

Comment:
```suggestion
  this->open(QString());
```
Don't use different semantics, even if it works. The `QString()` is probably what you want to express, like a null or empty string.